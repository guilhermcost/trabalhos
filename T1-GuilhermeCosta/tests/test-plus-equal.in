// deve retornar + e = 
a += b;

