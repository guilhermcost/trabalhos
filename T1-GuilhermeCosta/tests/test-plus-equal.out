(2,ID,"a")
(2,SYM,"+")
(2,SYM,"=")
(2,ID,"b")
(2,SYM,";")
